.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
I think we can remove the `.gitignore` file since it's only ignoring the editor-related folder.

Reply:
I can do that but I think it is good practice to have it.

Reply:
I prefer to keep `.gitignore` for project-related paths and let developer manage their development environment specific in their global ignore list.